### ISSUE TITLE
range operator followed by count operator crash

### ISSUE THREAD

**Issue by danylaporte** [Reactive-Extensions/RxJS#712]
```js
Rx.Observable.range(0, 10).count().subscribe(console.log.bind(console));
```
on the latest version (2.5.2)

**Actual**
throw Uncaught TypeError: Rx.Observable.range(...).count is not a function

in the browser and in nodejs.

**Expected**
10 written in the console.

**Comment by paulpdaniels** [comment]
Which library? rx.lite and rx don't include it by default.  You need rx.aggregates or rx.all

**Comment by danylaporte** [comment]
In node, I have installed using 
npm install rx

In the browser, I have used 
https://rawgit.com/Reactive-Extensions/RxJS/master/dist/rx.all.js

**Comment by paulpdaniels** [comment]
Problem was caused by an overriding member variable.  Fix is in #713.
